addons/a11y/src/components/A11YPanel.tsx

```diff
@@ -1,23 +1,22 @@
 import React, { Component, Fragment } from 'react';
-import PropTypes from 'prop-types';
 
 import { styled } from '@storybook/theming';
 
 import { STORY_RENDERED } from '@storybook/core-events';
 import { ActionBar, Icons } from '@storybook/components';
 
-import EVENTS from '../constants';
-
-import Tabs from './Tabs';
-import Report from './Report';
+import { AxeResults, Result } from 'axe-core';
+import { Report } from './Report';
+import { Tabs } from './Tabs';
+import { EVENTS } from '../constants';
 
 const Icon = styled(Icons)(
   {
     height: '12px',
     width: '12px',
     marginRight: '4px',
   },
-  ({ status, theme }) =>
+  ({ status, theme }: any) =>
```

Comment:
Yep! I was a bit lazy here because I think it should be automatically inferred by TS, I will investigate why it's not. I think we can do better typings in `@storybook/theming` ;)

Reply:
Ok, it looks like I can not do better for now, if I try a stronger typings I got TS errors elsewhere...